doc/nrf/app_dev/programming.rst

```diff
@@ -56,16 +56,35 @@ Programming without ``--erase``
 
      west flash
 
+  ..
+
+.. _readback_protection_error:
```

Comment:
Since now the ref link is moved, we would require to rephrase the sentence here: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16859/nrf/applications/nrf5340_audio/doc/building.html#running-the-script (note section. nrf\applications\nrf5340_audio\doc\building.rst - line no:114). 

I think its okay here: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16859/nrf/app_dev/device_guides/nrf53/building_nrf53.html#using-the-command-line

Reply:
Good catch, thanks.